tests/unit/cli_test_extensions.py

```diff
@@ -170,13 +184,17 @@ def command_test(self, desc, command_grp, inputs, exp_response, mock_files,
             and options that do not communicate with a server.  It is faster
             than installing the mock repository
 
-          condition (True, False, or 'pdb'):
+          condition (True, False, or 'pdb', or 'verbose'):
             If True, the test is executed.
             If False, the test is skipped.
             If 'pdb', the test breaks in the debugger.
+            If 'verbose' and verbose is None, verbose is set to True.
 
           verbose (:class:`py:bool`):
-            If `True` the assembled command line will be displayed
+            If `True` the assembled command line and the environment variables
+            set by 'env' will be displayed.
+            If `None`, verbose is set to True if condition=='verbose', otherwise
```

Comment:
I must be getting old.  This statement confuses me.

Reply:
I agree it is hard to read, but it is meant as written. None means that the condition argument determines the verbosity.

But maybe that is too complicated, and we should not have a value of None for verbose, and display in verbose mode if verbose == True or condition == 'verbose'. I'll change that.

Reply:
OK